settings.py

```diff
@@ -202,11 +204,20 @@
         }
     },
     'handlers': {
+        'sentry': {
+            'level': 'ERROR',
+            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
+            'tags': {'custom-tag': 'x'},
+        },
         'mail_admins': {
             'level': 'ERROR',
             'filters': ['require_debug_false'],
             'class': 'django.utils.log.AdminEmailHandler'
         },
+        'console': {
```

Comment:
Did you mean to add this also?

Reply:
Yea, it seems that raven/sentry need a dummy handler to point at